docs/nodes/devops-resources.md

```diff
@@ -0,0 +1,7 @@
+# DevOps Resources
+
+This section will be provide an overview of different available resources for Node Operators looking to test out Celestia architecture.
+
+## Docker Compose
+
+* One resource for playing with docker-compose for Celestia infrastructure is the [ephemeral-cluster](https://github.com/celestiaorg/ephemeral-cluster) repository. This repository provides infra-as-code specifications on running a docker-compose instance of Celestia devnet. It is not recommended to be used as-is for running nodes individually but can be used as a reference specification for anyone looking to modify it for their own purposes.
```

Comment:
I'm not sure we want to refer to the ephemeral cluster as a "reference specification". 
It's not targeted at node operators, it's a place to do integration testing for development.

Reply:
Check the new updated writing.